rllib/agents/a3c/a3c_tf_policy.py

```diff
@@ -142,7 +142,7 @@ def clip_gradients(
     grads_and_vars = optimizer.compute_gradients(
         loss, policy.model.trainable_variables()
     )
-    grads = [g for (g, v) in grads_and_vars]
+    grads = [g for (g, v) in grads_and_vars if g is not None]
     grads, _ = tf.clip_by_global_norm(grads, policy.config["grad_clip"])
     clipped_grads = list(zip(grads, policy.model.trainable_variables()))
```

Comment:
I guess my question here is:

1. grads_and_vars are computed by optimizer based on policy.model.trainable_variables()
2. instead of simply taking all the grads out, we do a little bit filter now.
3. right after the filtering, grads get zipped again with all the trainable_variables().
4. So how do we make sure the grads that are left still match the right trainable variables?